server/src/prisma/config/yupValidationPrismaClient.js

```diff
@@ -1,49 +1,29 @@
-import getValidation from "./getValidation.js";
 import { ValidationError } from "yup";
 
 const mutativeActions = ["create", "update", "upsert"];
 
-mutativeActions.map((action) => {});
+const customMutativeQueries = {}
+mutativeActions.forEach(action => {
+  customMutativeQueries[action] = async ({ model, operation, args, query }) => {
+    let transformedData;
+    try {
+      const { default: schema } = await import(`./../validations/${model}Schema.js`)
```

Comment:
consider a separate try/catch block for this line - I imagine students will forget to add the schema so we want to throw a different kind of error to disambiguate for them.